web-client/src/presenter/actions/TrialSession/extractUserNotesFromCalendaredCasesAction.ts

```diff
@@ -16,7 +16,7 @@ export const extractUserNotesFromCalendaredCasesAction = ({
   const userNotes = [];
 
   for (const calendaredCase of calendaredCases) {
-    userNotes.push(calendaredCase.notes);
+    userNotes.push((calendaredCase as { notes?: unknown }).notes);
```

Comment:
This still results in a TS2345 error. I think the correct fix is to annotate the correct type for `userNotes` on line 16. If you do so, you won't need this new assertion on line 19 either.